core/src/test/scala/org/apache/spark/SecurityManagerSuite.scala

```diff
@@ -112,6 +126,25 @@ class SecurityManagerSuite extends SparkFunSuite with ResetSystemProperties {
     securityManager.setViewAclsGroups("group4,group5")
     assert(securityManager.checkUIViewPermissions("user1") === false)
     assert(securityManager.checkUIViewPermissions("user2") === false)
+
+
+    val conf2 = new SparkConf
```

Comment:
I don't think we need this section of tests, the one above that uses BogusServiceProvider is enough.